mops.toml

```diff
@@ -17,7 +17,10 @@ test = "2.1.1"
 bench = "1.0.0"
 fuzz = "0.2.1"
 matchers = "https://github.com/kritzcreek/motoko-matchers#v1.3.1@5ba5f52bd9a5649dedf5e2a1ccd55d98ed7ff982"
-base-0-14-8 = "https://github.com/dfinity/motoko-base#moc-0.14.8@9e6010595cb9bad5b637ab7438fd11dc79a21a06"
+base-0-14-13 = "https://github.com/dfinity/motoko-base#moc-0.14.13@9e6010595cb9bad5b637ab7438fd11dc79a21a06"
```

Comment:
The hash does not need to change? Anyway, we probably don't need to bump this dependency